docs/getting-started-guides/dcos.md

```diff
@@ -0,0 +1,180 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/getting-started-guides/dcos.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+Getting started with Kubernetes on DCOS
+----------------------------------------
+
+This tutorial will walk you through setting up [Kubernetes-Mesos](https://github.com/mesosphere/kubernetes-mesos) on a [Datacenter Operating System (DCOS)](https://mesosphere.com/product/) cluster and verifying it with the [DCOS CLI](https://github.com/mesosphere/dcos-cli) and its [Kubectl plugin](https://github.com/mesosphere/dcos-kubectl).
+
+**Table of Contents**
+
+- [About Kubernetes on DCOS](#about-kubernetes-on-dcos)
+- [Resources](#resources)
+- [Prerequisites](#prerequisites)
+- [Install](#install)
+- [Uninstall](#uninstall)
+
+
+## About Kubernetes on DCOS
+
+The [DCOS](https://docs.mesosphere.com/) is a new kind of operating system that spans all of the machines in your datacenter or cloud and treats them as a single computer, providing a highly elastic and highly scalable way of deploying applications, services, and big data infrastructure on shared resources. The DCOS consists of everything necessary to build out a self-healing, fault-tolerant, and scalable solution.
```

Comment:
This feels a bit too much like marketing.  Please can you tone it down a touch?  
It would help to  just describe what it does, and drop the "new kind of operating system" blather!

Also the "everything" in the last sentence seems a little over-reaching.  Everything?  Really?